be/src/exec/spill/spiller.hpp

```diff
@@ -173,6 +176,7 @@ StatusOr<ChunkPtr> SpillerReader::restore(RuntimeState* state, TaskExecutor&& ex
     ASSIGN_OR_RETURN(auto chunk, _stream->get_next(_spill_read_ctx));
     RETURN_IF_ERROR(trigger_restore(state, std::forward<TaskExecutor>(executor), std::forward<MemGuard>(guard)));
     _read_rows += chunk->num_rows();
+    COUNTER_UPDATE(_spiller->metrics().restore_rows, chunk->num_rows());
```

Comment:
Spiller::restore also update restore_rows. is it as expected?

Reply:
> Spiller::restore also update restore_rows. is it as expected?

Spiller::restore need call SpillerReader::restore, and SpillableHashJoinProbeOperator may call SpillerReader::restore directly, so I remove the update in Spiller::restore(see line 94) and only update it in SpillerReader::restore